src/Endpoints/AbstractEndpoint.php

```diff
@@ -42,18 +41,20 @@ public function getCollectionName(): string
 
     protected function queryBuild(array $params): string
     {
-        if ($params) {
-            return '?' . Query::build($params);
+        $p = [];
+        foreach ($params as $k => $v) {
+            $p[] = urldecode($k) . "=" . urlencode($v);
         }
-        return '';
+
+        return '?' . implode('&', $p);
```

Comment:
Is not same like [http_build_query](https://www.php.net/manual/en/function.http-build-query.php) ?

Reply:
This is right. Let me change this with http_build_query()